src/App/App.test.tsx

```diff
@@ -1,22 +1,27 @@
-import { render, screen, fireEvent } from '@testing-library/react';
 import App from './App';
+import { render, screen, fireEvent, cleanup, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import { fetchAllCurrentUSAData, fetchCurrentStateData } from '../apiCalls'
-import { singleState, usaData } from '../sampleData';
+import { singleState, stateData, usaData } from '../sampleData';
 import '@testing-library/jest-dom';
+import { MemoryRouter } from 'react-router-dom';
+// const App = require('./App')
 jest.mock('../apiCalls');
  
 describe('App', () => {
+  let mockedUSAFetch = fetchAllCurrentUSAData as jest.MockedFunction<typeof fetchAllCurrentUSAData>
+  let mockedSingleFetch = fetchCurrentStateData as jest.MockedFunction<typeof fetchCurrentStateData>
   beforeEach(() => {
-    const mockedUSAFetch = fetchAllCurrentUSAData as jest.MockedFunction<typeof fetchAllCurrentUSAData>
-    const mockedSingleFetch = fetchCurrentStateData as jest.MockedFunction<typeof fetchCurrentStateData>
-
     mockedUSAFetch.mockResolvedValueOnce(usaData[0])
     mockedSingleFetch.mockResolvedValueOnce(singleState)
 
     render(
-      <App />
+      <MemoryRouter>
+        <App />
+      </MemoryRouter>
     );
   });
+  afterEach(()=> cleanup())
```

Comment:
Is this actually doing anything, did we determine?

Reply:
I'm not sure - will default to Joe on this. Let's look at this on the next testing branch.